financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/admin/impl/DefaultConsentAdminHandler.java

```diff
@@ -41,23 +37,39 @@
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.admin.model.ExternalAPIAdminConsentRevokeResponseDTO;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.admin.model.ExternalAPIAdminConsentSearchRequestDTO;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.admin.model.ExternalAPIAdminConsentSearchResponseDTO;
+import org.wso2.financial.services.accelerator.consent.mgt.extensions.admin.model.SearchTypeEnum;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.admin.utils.ConsentAdminUtils;
+import org.wso2.financial.services.accelerator.consent.mgt.extensions.admin.utils.ExternalAPIConsentAdminUtils;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.common.ConsentException;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.common.ConsentExtensionConstants;
-import org.wso2.financial.services.accelerator.consent.mgt.extensions.common.ExternalAPIUtil;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.common.ResponseStatus;
+import org.wso2.financial.services.accelerator.consent.mgt.extensions.common.model.ExternalAPIConsentResourceRequestDTO;
 import org.wso2.financial.services.accelerator.consent.mgt.extensions.internal.ConsentExtensionsDataHolder;
 import org.wso2.financial.services.accelerator.consent.mgt.service.ConsentCoreService;
 
 import java.util.ArrayList;
 import java.util.Map;
-import java.util.UUID;
 
 /**
  * Consent admin handler default implementation.
  */
 public class DefaultConsentAdminHandler implements ConsentAdminHandler {
     private static final Log log = LogFactory.getLog(DefaultConsentAdminHandler.class);
+    ConsentCoreService consentCoreService;
+    boolean isExtensionsEnabled;
+    boolean isExternalPreConsentRevocationEnabled;
+    boolean isExternalPreConsentSearchEnabled;
+
+    public DefaultConsentAdminHandler() {
+
+        FinancialServicesConfigParser configParser = FinancialServicesConfigParser.getInstance();
+        consentCoreService = ConsentExtensionsDataHolder.getInstance().getConsentCoreService();
+        isExtensionsEnabled = configParser.isServiceExtensionsEndpointEnabled();
+        isExternalPreConsentRevocationEnabled = configParser.getServiceExtensionTypes()
+                .contains(ServiceExtensionTypeEnum.PRE_PROCESS_CONSENT_REVOKE);
+        isExternalPreConsentSearchEnabled = configParser.getServiceExtensionTypes()
```

Comment:
Shall we rename this variable `isExternalPostConsentSearchEnabled` instead of `isExternalPreConsentSearchEnabled` since this is response alteration

Reply:
corrected